forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -4653,53 +4656,54 @@ public final void addChangedCardKeywords(final List<String> keywords, final List
         }
     }
 
-    public final KeywordInterface getKeywordForStaticAbility(String kw, final long staticId) {
+    public final KeywordInterface getKeywordForStaticAbility(String kw, final long staticId, final long idx) {
         KeywordInterface result;
-        if (staticId < 1 || !storedKeywords.contains(staticId, kw)) {
+        Triple<String, Long, Long> triple = Triple.of(kw, staticId, idx);
+        if (staticId < 1 || !storedKeywords.containsKey(triple)) {
             result = Keyword.getInstance(kw);
             result.setStaticId(staticId);
             result.createTraits(this, false);
             if (staticId > 0) {
-                storedKeywords.put(staticId, kw, result);
+                storedKeywords.put(triple, result);
             }
         } else {
-            result = storedKeywords.get(staticId, kw);
+            result = storedKeywords.get(triple);
         }
         return result;
     }
 
     public final void addKeywordForStaticAbility(KeywordInterface kw) {
         if (kw.getStaticId() > 0) {
-            storedKeywords.put(kw.getStaticId(), kw.getOriginal(), kw);
+            storedKeywords.put(Triple.of(kw.getOriginal(), kw.getStaticId(), 0l), kw);
```

Comment:
@tool4ever can you test if that is good enough? like with our friend Henzi Toolbox?

Reply:
Anything specific? He seems to still work from what I can tell

Reply:
because i didn't copy the idx, i thought it might get duplicated entities (and resulting in two Blitz trigger?)